### PR TITLE
Add JSON formart standard.

### PR DESCRIPTION
**Pull request by HouCoder**
We decided to follow the RPC response format standard in Ajax requests, if you want to get the related conversations, please Slack me, I'll send the links to you.

## Summary of changes

- Add JSON formart standard.

## How to Test

Testing link: https://github.com/HouCoder/style-guide/blob/standardized-json-response/language-json.md

/fyi: @juwai/all Please review.

### CODE REVIEW

File: language-json.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# JavaScript Object Notation
+
+AJAX response have to follow [RPC convention](http://docs.juwai.io/rpc/property.html#module-jw.property.views.rpc).
```
Comment (tjoelsson) [inline language-json.md]: That's a link to our internal documentation

Comment (HouCoder) [inline language-json.md]: ok, I'll remove it, I tested this link before with different IP addresses, I thought it is accessible for all public IP addresses, so I linked it in the documentation.

**Review by mark-juwai — APPROVED** [review]

**Comment by tjoelsson** [comment]
@ben-juwai please review

**Comment by HouCoder** [comment]
Closing this dead PR since we decide to follow REST standards in our projects.

**Pull request closed without merging**